source/features/improve-shortcut-help.tsx

```diff
@@ -48,11 +47,18 @@ function observeShortcutModal({key, target}: KeyboardEvent): void {
 	}
 }
 
-function init(): void {
+function init(): Deinit[] {
 	document.addEventListener('keypress', observeShortcutModal);
+
+	return [
+		observer,
+		() => {
+			document.removeEventListener('keypress', observeShortcutModal);
+		},
+	];
 }
 
 void features.add(import.meta.url, {
 	awaitDomReady: false,
-	init: onetime(init),
+	init,
```

Comment:
I wonder if it's worth de-initifying features that act on every page and only register a global event listener.

Maybe in this case `onetime(init)` is better, since it won't mean destroying and recreating the listener on every pjax navigation.

Reply:
Agreed

Reply:
I restored the `onetime(init)` for all relevant unconditional features, including the ones which create an observer (for the same reason as stated above).

I also added a `onetime` to two features which I think were missing one:
 - `linkify-user-location`
 - `reload-failed-proxied-images`